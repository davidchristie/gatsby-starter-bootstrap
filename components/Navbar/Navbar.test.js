import React from 'react'
import renderer from 'react-test-renderer'

import Navbar from './Navbar'

it('renders with errors', () => {
  renderer.create(<Navbar />)
})

it('matches render snapshot', () => {
  const tree = renderer.create(<Navbar />).toJSON()
  expect(tree).toMatchSnapshot()
})
