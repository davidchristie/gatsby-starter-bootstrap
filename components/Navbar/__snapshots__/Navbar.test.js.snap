// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches render snapshot 1`] = `
<nav
  className="navbar sticky-top navbar-toggleable-sm navbar-inverse bg-danger"
>
  <button
    aria-controls="navbarColor02"
    aria-expanded="false"
    aria-label="Toggle navigation"
    className="navbar-toggler navbar-toggler-right collapsed"
    data-target="#navbarColor02"
    data-toggle="collapse"
    type="button"
  >
    <span
      className="navbar-toggler-icon"
    />
  </button>
  <div
    className="container"
  >
    <a
      className="text-center"
      onClick={[Function]}
      style={Object {}}
    >
      <h1
        className="navbar-brand mb-0"
      />
    </a>
    <div
      aria-expanded="false"
      className="navbar-collapse collapse"
      id="navbarColor02"
    >
      <ul
        className="navbar-nav mr-auto"
      >
        <li
          className="nav-item"
        >
          <a
            className="nav-link"
            onClick={[Function]}
            style={Object {}}
          >
            Home
          </a>
        </li>
        <li
          className="nav-item"
        >
          <a
            className="nav-link"
            onClick={[Function]}
            style={Object {}}
          >
            Profile
          </a>
        </li>
        <li
          className="nav-item dropdown"
        >
          <a
            aria-expanded="false"
            aria-haspopup="true"
            className="nav-link dropdown-toggle"
            data-toggle="dropdown"
            id="navbarDropdownMenuLink"
          >
            Link
          </a>
          <div
            aria-labelledby="navbarDropdownMenuLink"
            className="dropdown-menu bg-danger"
          >
            <a
              className="dropdown-item"
              href="https://github.com/jaxx2104/"
            >
              Github
            </a>
            <a
              className="dropdown-item"
              href="https://twitter.com/jaxx2104"
            >
              Twitter
            </a>
            <a
              className="dropdown-item"
              href="https://www.facebook.com/futoshi.iwashita"
            >
              Facebook
            </a>
          </div>
        </li>
      </ul>
    </div>
  </div>
</nav>
`;
